services/version.js

```diff
@@ -66,11 +66,9 @@ function latestDottedVersion(versions) {
   return version
 }
 
-// Given a list of versions (as strings), return the latest version.
-// Return undefined if no version could be found.
-function latest(versions, { pre = false } = {}) {
+function latestMaybeSemVer(versions, pre) {
```

Comment:
Ideas for a better name welcome :D

Reply:
I was just chuckling at this :laughing: 

I like it, and it gets my vote!